cypress/integration/common/priority_queue.spec.js

```diff
@@ -128,70 +120,45 @@ describe("TA Priority Queue", () => {
   });
 
   describe("Requeue Student", () => {
-    it("TA requeues student and student leaves the queue", function () {
-      cy.get("body").should("contain", "Help Next");
-      cy.get("button").contains("Help Next").click();
-
-      cy.get(
-        `[data-cy="list-helping"] [data-cy="queue-list-item-${this.question.id}"]`
-      )
-        .should("be.visible")
-        .click();
+    it.only("Requeue student works", function () {
```

Comment:
Not a big deal cause there's only one test in the describe but don't think you want the `it.only` here